src/views/IngredientsView.vue

```diff
@@ -39,12 +39,20 @@ const getIngredients = async code => {
   if (!ingredients_text_from_image) {
     return null;
   }
-  const {
+  if(IS_DEVELOPMENT_MODE){
+    const {
+    data: { corrected, corrections }
+    }
```

Comment:
I didn't want to break any code and I couldn't find where the IngredientsView.vue is being used to check whether the variable data is being used anywhere or not. Also in this post request, the response is getting stored in a variable called data, so to prevent any breaks in the further code, I created a variable with the same name. Should I remove it?

Reply:
Nice catch, I did not think about breaking the answer. 

To be honest, this code is probably not used, because I never finished the ingredient games by lack of time

Reply:
Okay. Then I'll remove it I think since it's not being used currently.